concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -1514,6 +1514,31 @@ jobs:
 
                   cf push
 
+        - task: update-kibana-timezone
+          config:
+            platform: linux
+            image: docker:///ruby#2.2-slim
```

Comment:
@mtekel Thanks. This PR will be merged first. @keymon knows to do the rebasing of #381